README.md

```diff
@@ -90,58 +90,63 @@ function SearchBar() {
 }
 ```
 
-## `AnswersHeadlessContext`
-### Class Components
+## Class Components
 
-For users that want to use class components instead of functional components, you can use the `AnswersHeadlessContext` directly to dispatch actions and receive updates from state.
+For users that want to use class components instead of functional components, you can use the `SearchHeadlessContext` directly to dispatch actions, and the `subscribeToStateUpdates` HOC to receive updates from state.
 
-As an example, here is our simple SearchBar again, rewritten as a class using `AnswersHeadlessContext`.
+These also work with functional components.
+
+## `subscribeToStateUpdates`
+
+Here is our MostRecentSearch component again, rewritten as a class with `subscribeToStateUpdates`.
 
 ```tsx
-import { AnswersHeadlessContext, AnswersHeadless, State } from '@yext/answers-headless-react';
+import { subscribeToStateUpdates } from '@yext/search-headless-react';
 import { Component } from 'react';
 
-export default class Searcher extends Component {
-  static contextType = AnswersHeadlessContext;
-  unsubscribeQueryListener: any;
-  state = { query: "" };
-
-  componentDidMount() {
-    const answers: AnswersHeadless = this.context;
-    this.unsubscribeQueryListener = answers.addListener({
-      valueAccessor: (state: State) => state.query.mostRecentSearch,
-      callback: newPropsFromState => {
-        this.setState({ query: newPropsFromState })
-      }
-    });
-  }
+interface Props {
+  mostRecentSearch: string
+}
 
-  componentWillUnmount() {
-    this.unsubscribeQueryListener();
+class MostRecentSearch extends Component<Props> {
+  render() {
+    return <div>Showing results for {this.props.mostRecentSearch}</div>;
   }
+}
+
+export default subscribeToStateUpdates(state => ({
```

Comment:
we want the updated README portion here from develop, I think you might have reverted back when resolving merge conflict